ReactSkia/components/RSkComponentImage.h

```diff
@@ -1,11 +1,12 @@
 #pragma once
 
 #include "ReactSkia/components/RSkComponent.h"
+#include "ReactSkia/views/common/RSkDrawUtils.h"
 
 namespace facebook {
 namespace react {
 
-class RSkComponentImage final : public RSkComponent {
+class RSkComponentImage final : public RSkComponent , public RSkDrawUtils{
```

Comment:
Is the purpose to use inheritance here simply to call RSkDrawUtils functions without namespace?
Is is possible to have `using RSkDrawUtils` instead of inheritance here?
Moreover, if moving the using statement into c++ file, this could save one more header dependency.
I.e. you could also move `#include "ReactSkia/views/common/RSkDrawUtils.h"` into cpp.

Reply:
will work on it & update.